docs/user-guide/stopwords.rst

```diff
@@ -0,0 +1,179 @@
+Stop Words in Text Processing
+=============================
+
+Stop words are common words that are often filtered out in natural language processing (NLP) tasks because they typically don't carry significant meaning. Examples in English include "the," "is," "at," "which," and "on." These words appear so frequently in language that they can distort text processing tasks if not properly managed. NVIDIA NeMo Curator provides built-in stop word lists for several languages to support text analysis and extraction processes.
+
+Key characteristics of stop words:
+
+* They appear with high frequency in text
+* They typically serve grammatical rather than semantic functions
+* They're language-specific (each language has its own set of stop words)
+* Removing them can improve efficiency in many NLP tasks
+
+Why Stop Words Matter in NeMo Curator
+------------------------------------
+
+In NeMo Curator, stop words play several important roles:
+
+* **Text Extraction**: The text extraction process (especially for Common Crawl data) uses stop word density as a key metric to identify meaningful content
+* **Language Detection**: Stop words help in language detection and processing
+* **Efficient Processing**: Filtering stop words reduces the amount of data that needs to be processed
+* **Boilerplate Removal**: Stop word density helps differentiate between main content and boilerplate in web pages
+
+Available Stop Word Lists
+------------------------
+
+NeMo Curator includes built-in stop word lists for the following languages:
```

Comment:
This table is not accurate - we support all the languages accessible by `justext.get_stoplist`, which are listed here: https://github.com/miso-belica/jusText/tree/main/justext/stoplists

Since jusText does not have stoplists for Thai, Japanese, or Chinese, we have the lists in Curator.